cf-agent/verify_packages.c

```diff
@@ -2798,7 +2803,8 @@ static bool ExecuteSchedule(EvalContext *ctx, const PackageManager *schedule, Pa
                             const char *sp = PrefixLocalRepository(a.packages.package_file_repositories, pi->name);
                             if (sp != NULL)
                             {
-                                strcpy(offset, sp);
+                                strlcpy(offset, sp,
+                                        estimated_size - command_len);
```

Comment:
No reason to split this into two lines while there's a much longer line right above, I think.